eb_hooks.py

```diff
@@ -734,6 +734,10 @@ def pre_configure_hook_LAMMPS_zen4(self, *args, **kwargs):
                 if cpu_target == CPU_TARGET_ZEN4:
                     # There is no support for ZEN4 in LAMMPS yet so falling back to ZEN3
                     self.cfg['kokkos_arch'] = 'ZEN3'
+                if cpu_target == CPU_TARGET_SAPPHIRE_RAPIDS:
+                    # These version do support Sapphire Rapids, but the easyblock does not contain the mapping yet
```

Comment:
Can we open a PR to the LAMMPS easyblock instead, and use that?

I would definitely prefer that approach along with `--include-easyblocks-from-commit`...

Reply:
Yes, I was going to do that, see https://gitlab.com/eessi/support/-/issues/68#note_2318954511. However, it's tricky for the older version, as its archspec version does not identify the CPU correctly. The hook is just an easy and quick solution for both problems.

Reply:
Done in https://github.com/easybuilders/easybuild-easyblocks/pull/3569 and https://github.com/EESSI/software-layer/pull/901/commits/732e6d423c1b253582b6619d4520d956b0323634. We still need a fix (and rebuild) for the older version, though.

Reply:
You can explicitly tell the Lammps build what cpu arch to use, so a `try-amend` might be enough for the older case

Reply:
Hmm, that's not global so would still require a parse hook, so no better than the previous approach